README.md

```diff
@@ -72,12 +72,11 @@ The available `dx` functions are
 
 ### from_tables Actions
 
-After a `apply_sql` or `unpivot_string_columns` command, you can apply the following actions:
+After a `with_sql` or `unpivot_string_columns` command, you can apply the following actions:
 
 * `explain` explains the queries that would be executed
-* `execute` executes the queries and shows the result in a unioned dataframe
-* `to_union_dataframe` unions all the dataframes that result from the queries
-
+* `display` executes the queries and shows the result in a unioned dataframe
```

Comment:
display() would only output a limited number or rows correct? should we add that information?

Reply:
Updated